crates/client/rpc/src/errors.rs

```diff
@@ -19,28 +19,30 @@ pub enum StarknetRpcApiError {
     InvalidTxnIndex = 27,
     #[error("Class hash not found")]
     ClassHashNotFound = 28,
-    #[error("Class already declared")]
-    ClassAlreadyDeclared = 51,
     #[error("Requested page size is too big")]
     PageSizeTooBig = 31,
     #[error("There are no blocks")]
     NoBlocks = 32,
     #[error("The supplied continuation token is invalid or unknown")]
     InvalidContinuationToken = 33,
+    #[error("Too many keys provided in a filter")]
+    TooManyKeysInFilter = 34,
+    #[error("Failed to fetch pending transactions")]
+    FailedToFetchPendingTransactions = 38,
     #[error("Contract error")]
     ContractError = 40,
     #[error("Invalid contract class")]
     InvalidContractClass = 50,
-    #[error("Too many storage keys requested")]
-    ProofLimitExceeded = 10000,
-    #[error("Too many keys provided in a filter")]
-    TooManyKeysInFilter = 34,
+    #[error("Class already declared")]
+    ClassAlreadyDeclared = 51,
+    #[error("The transaction version is not supported")]
+    UnsupportedTxVersion = 61,
```

Comment:
Did you choose those numbers yourself?

Reply:
Comes from:
https://github.com/starkware-libs/starknet-specs/blob/0e859ff905795f789f1dfd6f7340cdaf5015acc8/api/starknet_write_api.json#L272

Reply:
maybe mention it in a comment